pandas/io/json.py

```diff
@@ -123,32 +123,39 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
         file. For file URLs, a host is expected. For instance, a local file
         could be ``file://localhost/path/to/table.json``
 
-    orient
+    orient : string, indicating the expected format of the JSON input.
```

Comment:
Can you put the explanation on the next line? (but leave the type (so 'string') on this one)